### DISCUSSION TITLE
Can't resolve '~ng-zorro-antd/style/index.min.css' in '/Users/Desktop/filter/filter-ui/src'

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [NG-ZORRO/ng-zorro-antd#8121] [Q&A] [answered]
Hello everyOne,
I keep getting the error : 
![Capture d’écran 2023-10-16 à 09 54 19](https://github.com/NG-ZORRO/ng-zorro-antd/assets/101296742/524f8ae9-455b-4eaa-9bf6-b00a27dc344c)
this is my styles.scss
![Capture d’écran 2023-10-16 à 09 54 59](https://github.com/NG-ZORRO/ng-zorro-antd/assets/101296742/4568f5b3-5033-4bdb-8f6b-b6a664956498)

**Comment by ParsaArvanehPA** [comment]
Are you still having this problem? Or you managed to fix it yourself?

**Reply by Elmatbaisaad** [reply]
Yes :/

**Reply by KeivinIsmaili** [reply, marked as answer]
@Elmatbaisaad  You are trying to use scss configuration meanwhile ng-zorro supports only css and less
